internal/packager/packing.go

```diff
@@ -79,74 +79,6 @@ func PackInvocationImageContext(cli command.Cli, app *types.App, target io.Write
 	return nil
 }
 
-// Pack packs the app as a single file
```

Comment:
Were these functions unused?

Reply:
They were totally unused yes.

Reply:
They were used as experimental commands, but IIRC the commands were removed.